internal/functions/Invoke-Program.ps1

```diff
@@ -164,7 +164,10 @@ function Invoke-Program {
         }
 
         Write-Message -Level Debug -Message "Acceptable success return codes are [$($SuccessReturnCode -join ',')]"
-        Write-Message -Level Verbose -Message "Starting process [$Path] with arguments [$ArgumentList] on $ComputerName through $Authentication protocol"
+
+        $secureArgumentList = "$ArgumentList".Replace($($Credential.GetNetworkCredential().Password), "****************")
```

Comment:
```suggestion
                        $secureArgumentList = $ArgumentList -replace '(PASSWORD|SAPWD)=".*', '$1="********"'
```

Reply:
@andreasjordan please confirm my github suggestion of your comment suggestion 🙇🏼

Reply:
Yes, but I think there a too many blanks.

Reply:
ill commit and see if tests fail 😊